packages/api/src/shared/shared.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
```

Comment:
I'm afraid that creating a module named Shared brings the risk it would be used for everything. I see you used it for configuration only so maybe  ConfigModule would be a better idea?

Also, I would consider making such a module global.

Reply:
I agree with Konrad. Sometimes Shared module is reasonable, but not in this case. 
We should prefer domain slices (vertical) instead of horizontal (technical) slices in our application architecture. 
So module per domain / feature. In this case better solution will be GithubClientConfigService / module etc. 

Then if you want to change something with Github changes will be introduced only in one module. 
Furthermore, let's imagine you want to remove possibility of GitHub login - then you remove a whole module and others remain the same. It's Open-Closed from SOLID in practice. Concerns which changes together should be coupled in some module.